shared/src/convention/convention.schema.ts

```diff
@@ -610,7 +616,9 @@ const addIssueIfSundayIsInSchedule = (
   addIssue: (message: string, path: string) => void,
   conventionId: ConventionId,
   complexSchedule: DailyScheduleDto[],
+  dateEnd: DateString,
 ) => {
+  if (isConventionOld(dateEnd)) return;
```

Comment:
Peut être uniquement si statut fin de parcours (validé/obsolète/abandonnée/refusée) mais les conventions qui pourraient être en milieu de parcours faudrait inclure le check des heures.

Ce n'est pas un sujet tech uniquement mais aussi métier.

Reply:
Je veux bien faire cette vérif, mais normalement on a pas de conventions "vieille" sur des statuts intérmédiaire, car on les passe en Obsolète quand ils dépassent 2 mois

Reply:
OK suite à cet échange

------------------------------

Jérôme — Aujourd’hui à 14:43
yes
je confirme :
select count(*)
from conventions
where date_end < '2024-08-31'
  and status not in ( 'ACCEPTED_BY_VALIDATOR', 'REJECTED', 'CANCELLED', 'DEPRECATED')
Done 0
Benjamin — Aujourd’hui à 14:43
Ok propre car j'avais peur de la situations des conventions créées bien en amont de la date de démarrage
Jérôme — Aujourd’hui à 14:43
(sur 290125 au total)
Benjamin — Aujourd’hui à 14:43
![image](https://github.com/user-attachments/assets/15270121-4a76-41bc-a323-6fb9ef52352e)
Si au final elles atteignent un statut de convention non éditable c'est bon.
Et si il y a eu d'anciennes conventions qui sont passées avec des planning non valides c'était au prescripteur de contrôler